paparazzi/src/main/java/app/cash/paparazzi/PaparazziSdk.kt

```diff
@@ -182,6 +183,15 @@ public class PaparazziSdk @JvmOverloads constructor(
     }
 
     bridgeRenderSession = createBridgeSession(renderSession, renderSession.inflate())
+
+    val viewGroup = bridgeRenderSession.rootViews[0].viewObject as ViewGroup
+    // Fixes issue where layoutlib's [DisplayManagerGlobal] doesn't have a [registerForRefreshRateChanges] method.
+    // This method is called by [Display.getRefreshRate] which is called by [AndroidPrefetchScheduler.calculateFrameIntervalIfNeeded].
+    // [Display.getRefreshRate] doesn't call [registerForRefreshRateChanges] if [mRefreshRateChangesRegistered] is true.
+    Display::class.java.getDeclaredField("mRefreshRateChangesRegistered").apply {
+      isAccessible = true
+      set(viewGroup.display, true)
+    }
```

Comment:
interesting, what changes led to this?  got link/error message to explain why this is needed?

Reply:
could it be related to either/both of these:

- https://android.googlesource.com/platform/frameworks/layoutlib/+/0602392919a5beb7021e98a8c54e8bc6bc5a176b%5E%21/#F0
- https://android.googlesource.com/platform/frameworks/layoutlib/+/76913ff05febc32526437dbbcf3598515c1e1e49%5E%21/#F0

Reply:
Compose 1.9.0 calls `Display.getRefreshRate()` which in turn calls `DisplayManagerGlobal.getInstance().registerForRefreshRateChanges()` which doesn't exist in layoutlib 16.0.0 version of `DisplayManagerGlobal`.